lib/mu-maildir.c

```diff
@@ -779,12 +779,24 @@ get_new_basename (void)
 				g_get_host_name ());
 }
 
+static const char*
+get_path_separator(const char *path)
+{
```

Comment:
Regarding tabs, I do not see any guideline or configuration. I am using Emacs. 8 spaces tabs? 4 spaces?

Regarding static char*, the input is constant and hence I thought it would be more consistent to return the same type of output.

Reply:
There are some guidelines, but the usually best to follow whatever file you're editing. `M-x c-guess` can be useful. For tabs, the width is just a cosmetic (local) matter. I use 8.

As for the `static char*`, we're writing to it later (and cast away the `const`, which I'm not a fan of)